### ISSUE TITLE
False positive DAR201

### ISSUE THREAD

**Issue by cthoyt** [terrencepreilly/darglint#109]
```python
import os
from subprocess import CalledProcessError, check_output  # noqa: S404


def get_git_hash() -> str:
    """Get the PyKEEN git hash.

    :return: The git hash, equals 'UNHASHED' if encountered CalledProcessError, signifying that the
     code is not installed in development mode.
    """
    with open(os.devnull, 'w') as devnull:
        try:
            ret = check_output(  # noqa: S603,S607
                ['git', 'rev-parse', 'HEAD'],
                cwd=os.path.dirname(__file__),
                stderr=devnull,
            )
        except CalledProcessError:
            return 'UNHASHED'
        else:
            return ret.strip().decode('utf-8')[:8]
```
Gives `test.py:get_git_hash:5: DAR201: - return`

Interestingly, removing the code makes darglint happy. The following passes:

```python
import os
from subprocess import CalledProcessError, check_output  # noqa: S404


def get_git_hash() -> str:
    """Get the PyKEEN git hash.

    :return: The git hash, equals 'UNHASHED' if encountered CalledProcessError, signifying that the
     code is not installed in development mode.
    """
    ...
```

**Comment by terrencepreilly** [comment]
The problem is with your indentation.  The second line should be indented 4 spaces.  (Or, whatever you've set your indentation to.)

For Google style strings, darglint emits an indentation error.  It should probably do the same here.

**Comment by cthoyt** [comment]
You mean the blank line should have whitespace? That's contra to flake8's reccomendations

**Comment by cthoyt** [comment]
Or do you mean the line that starts with "code"?

**Comment by terrencepreilly** [comment]
Sorry, I missed this.  Yes, the line starting with "code" should be indented.

**Comment by cthoyt** [comment]
Oh I see. There's that extra space there. Somehow I got in the habit of doing that to make longer blocks of documentation readable and never thought about it again because Sphinx doesn't complain
